Dockerfile.cli

```diff
@@ -0,0 +1,16 @@
+FROM php:7.0.26-cli-jessie
+RUN docker-php-ext-install pcntl
+RUN apt-get update && apt-get install -y git-core zip
+# try apt-get clean
+
+# replace with multi-stage build using the composer official image?
+RUN curl https://getcomposer.org/download/1.5.2/composer.phar -o /usr/local/bin/composer && chmod +x /usr/local/bin/composer 
```

Comment:
`1.5.2` is very specific.

Reply:
Should we port https://github.com/elifesciences/builder-base-formula/blob/master/elife/composer.sls#L53-L71 here or is there an updated .phar URL?

Reply:
That's the recommended way to install. AFAIK there's no `1.x` URI (maybe we could request one?).